### PR TITLE
Switch to GH forms for issues

### PR DESCRIPTION
**Pull request by CyberDem0n**
and make link to #patroni channel on PostgreSQL Slack more visible

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  id: expected
+  attributes:
+    label: What did you expect to happen?
+  validations:
+    required: true
+
+- type: textarea
+  id: environment
+  attributes:
+    label: Patroni/PostgreSQL/DCS version (please run commands below)
```
Comment (hughcapet) [inline .github/ISSUE_TEMPLATE/bug_report.yml:38]: what are the "commands below"?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- type: textarea
+  id: postgresLogs
+  attributes:
+    label: PostgreSQL log files
+    description: Please copy and paste any relevant PostgreSQL log output. This will be automatically formatted into code, so no need for backticks.
+    render: shell
+  validations:
+    required: true
+
+- type: input
```
Comment (hughcapet) [inline .github/ISSUE_TEMPLATE/bug_report.yml:82]: maybe just a checkbox here?

Comment (CyberDem0n) [reply .github/ISSUE_TEMPLATE/bug_report.yml:82]: I tried, it allows submitting an issue even is checkbox isn't checked :(

Comment (hughcapet) [reply .github/ISSUE_TEMPLATE/bug_report.yml:82]: like this?

```
options:
  - label: 'yes'
    required: true

**Comment by hughcapet** [comment]
👍

**Pull request merged**